<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>723</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>680</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>680</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="button2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="button6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="button4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="button9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="button1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="button0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonVolUp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Vol+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonVolDown">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Vol-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buttonChanUp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>chan+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonChenDown">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>chan-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelVol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 85, 0);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>0%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelChan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(170, 255, 127);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add0()</slot>
  <slot>add1()</slot>
  <slot>add2()</slot>
  <slot>add3()</slot>
  <slot>add4()</slot>
  <slot>add5()</slot>
  <slot>add6()</slot>
  <slot>add7()</slot>
  <slot>add8()</slot>
  <slot>add9()</slot>
 </slots>
</ui>
